app/backend/gunicorn.ocp4.cfg

```diff
@@ -0,0 +1,52 @@
+# Gunicorn configuration file.
+
+# Worker processes
+#
+#   workers - The number of worker processes that this server
+#       should keep alive for handling requests.
+#
+#       A positive integer generally in the 2-4 x $(NUM_CORES)
+#       range. You'll want to vary this a bit to find the best
+#       for your particular application's work load.
+#
+#   worker_class - The type of workers to use. The default
+#       sync class should handle most 'normal' types of work
+#       loads. You'll want to read
+#       http://docs.gunicorn.org/en/latest/design.html#choosing-a-worker-type
+#       for information on when you might want to choose one
+#       of the other worker classes.
+#
+#       A string referring to a Python path to a subclass of
+#       gunicorn.workers.base.Worker. The default provided values
+#       can be seen at
+#       http://docs.gunicorn.org/en/latest/settings.html#worker-class
+#
+#   worker_connections - For the eventlet and gevent worker classes
+#       this limits the maximum number of simultaneous clients that
+#       a single process can handle.
+#
+#       A positive integer generally set to around 1000.
+#
+#   timeout - If a worker does not notify the master process in this
+#       number of seconds it is killed and a new worker is spawned
+#       to replace it.
+#
+#       Generally set to thirty seconds. Only set this noticeably
+#       higher if you're sure of the repercussions for sync workers.
+#       For the non sync workers it just means that the worker
+#       process is still communicating and is not tied to the length
+#       of time required to handle a single request.
+#
+#   keepalive - The number of seconds to wait for the next request
+#       on a Keep-Alive HTTP connection.
+#
+#       A positive integer. Generally set in the 1-5 seconds range.
+#
+
+# workers = 18
```

Comment:
Should this be commented out? What is the default worker count?

Reply:
I did some digging in the repo and `backend.dc.json` sets this through an environment variable to 4 workers. I don't know what the ideal settings are (especially when using gevent workers)... I do remember we had issues with too many workers in the past.

This is the same config as it is now on GWELLS, except for this file doesn't use `chdir = 'backend'` . The newer container for OCP4 has a different folder structure so the chdir setting was leading to the wrong dir.

Reply:
I added a comment to this file to show that # workers is configured in the backend.dc.json file.